hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -15,6 +15,17 @@
   // 32-bit datawidth
   datawidth: "32",
 
+  // Power information for the design
+  power: {
+    // Power domains supported by the design
+    // Aon represents domain aon
+    // 0 represents domain 0
+    domains: ["Aon", "0"],
```

Comment:
Should we reverse the list for consistency? I.e., Aon is at index 1 in RTL...

Reply:
also, we already have the name `aon` in the clocks... should we use a slightly different naming scheme here to make the names unique? e.g. like `pwr_aon` / `pwr_d0` or `dom_aon` / `dom_0`?

Reply:
ah yes, i kept the names simple here because the selection that gets constructed ends up being `DomainXX`
I think you're right though..maybe adding `pwr` at least would be nice.

Reply:
wait sorry where are you seeing index 1? I actually always intended aon to be 0.
did i miss somewhere?

Reply:
oh then I misread something. nevermind then, this already seems to be aligned.